core/auth-crt/src/main/java/software/amazon/awssdk/authcrt/signer/internal/SigningUtils.java

```diff
@@ -15,6 +15,16 @@
 
 package software.amazon.awssdk.authcrt.signer.internal;
 
+import static software.amazon.awssdk.http.auth.aws.signer.SignerConstant.AUTHORIZATION;
```

Comment:
`SignerConstant` is marked `@SdkInternalApi` in `http-auth-aws` module, but is used here in `auth-crt` module, is that a concern?

Reply:
It's not in the feature branch ;) That change was made in consolidating PR 1 https://github.com/aws/aws-sdk-java-v2/pull/6480 

https://github.com/aws/aws-sdk-java-v2/blob/feature/master/technicaldebt-duplicatecode/core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/signer/SignerConstant.java#L21

Reply:
Oh right, I looked at master, my bad